### DISCUSSION TITLE
Paid Tasks for Core Contributions

### DISCUSSION THREAD

**Discussion by idimetrix** [elevanceit/elevanceit#1] [Announcements]
We offer a structured Bounty Program with tasks of varying difficulty levels: **major**, **middle**, and **minor**. 

Each task is assigned a specific **reward based on its complexity**, encouraging contributors to engage according to their skills and interests. 

**Complete tasks, earn bounties, and grow with our project!**

**Comment by BaseMax** [comment]
Hi @idimetrix 
I came across the announcement for the structured Bounty Program and find it exciting. I’m an experienced developer with over 10 years in software engineering, actively contributing to open-source projects, and I’m always looking for opportunities to collaborate and grow within innovative communities like yours.

Could you please share more details about the program?

Many thanks
